provider/resource_keycloak_openid_client_test.go

```diff
@@ -2095,14 +2144,50 @@ func testKeycloakOpenidClient_import(clientId string, enabled bool) string {
 data "keycloak_realm" "realm" {
 	realm = "%s"
 }
-
+import {
+	id = "${data.keycloak_realm.realm.id}/%s"
+	to = keycloak_openid_client.client
+}
 resource "keycloak_openid_client" "client" {
 	client_id   = "%s"
 	realm_id    = data.keycloak_realm.realm.id
 	access_type = "PUBLIC"
 	root_url    = ""
 	enabled     = %t
-	import      = true
 }
-	`, testAccRealm.Realm, clientId, enabled)
+	`, testAccRealm.Realm, clientId, clientId, enabled)
+}
+
+func testKeycloakOpenidClient_import_postcondition(clientId string, enabled bool) string {
```

Comment:
an example is taken from https://github.com/keycloak/terraform-provider-keycloak/issues/1007#issuecomment-2432989685